### DISCUSSION TITLE
¿Cómo puedo ejecutar desde VSC?

### DISCUSSION THREAD

**Discussion by Jessz3** [hackatomik/SISCOx256#33] [Preguntas Técnicas | Lab]
Traté de ejecutar el Knock Knock Server desde VSC y no me sale ningún error. El problema es que a pesar de que no me salgan errores, no puedo ver cómo se está ejecutando el programa. ¿Qué tendría que hacer?

**Comment by quantumquirkxyz** [comment]
1. **Abre la Terminal en VSC:**
   - Ve a **"Ver"** > **"Terminal"** en el menú superior, o usa el atajo **Ctrl + `** (la tecla de tilde grave, generalmente situada justo debajo de la tecla Esc).

2. **Navega a la Carpeta del Proyecto:**
   - Usa comandos de terminal para moverte a la carpeta donde está el archivo del Knock Knock Server. Por ejemplo:
     ```sh
     cd ruta/a/tu/carpeta
     ```

3. **Ejecuta el Programa:**
   - Dependiendo de cómo esté escrito el Knock Knock Server, usa el comando apropiado para ejecutarlo. Por ejemplo, si es un archivo Python:
     ```sh
     python nombre_del_archivo.py
     ```
     Si es un archivo Node.js:
     ```sh
     node nombre_del_archivo.js
     ```

4. **Verifica la Salida en la Terminal:**
   - La salida del programa, incluyendo cualquier mensaje o errores, aparecerá en la terminal de VSC. Asegúrate de que no haya mensajes de error y que el programa esté funcionando como se espera.

5. **Configuración de Depuración (Opcional):**
   - Si quieres ejecutar y depurar el programa en un entorno más controlado, configura una **"Configuración de depuración"**. Ve a **"Ejecutar"** > **"Agregar configuración"** en el menú lateral para crear un archivo de configuración `launch.json` que te permita definir cómo ejecutar el programa, si es necesario.

**Reply by quantumquirkxyz** [reply]
Esto es solo si usaste python en VS Code, en tal caso que no usaste python, ponte en contacto conmigo

**Reply by Jessz3** [reply]
En este caso utilicé java

**Reply by quantumquirkxyz** [reply]
Si ese es el caso, entonces necesitas descargar un compilador de java o una extensión de Java par Vs Code

https://youtu.be/5voE8tvtVV8?si=ftj9cf0EHv3PI8YT
